### PR TITLE
Image upload in Add Organization form

### PR DESCRIPTION
**Pull request by carlosbflores**
Changes on the Add Organization form, for uploading organization logo image.
Changes on the Organization card template for displaying the organization logo.

### CODE REVIEW

**Comment by carlosbflores** [comment]
Removed all Javascript code from template to Backbone view. DOM access changed to use jQuery selectors. And added some minor CSS fixes for logo image display.

File: src/app/organizations/add/template.hbs [hunk 1]
```diff
@@ -40,13 +40,13 @@
...
                 <div class="form-group">
-                  <input value="{{organization.description}}" type="text" class="input-bg-gray form-control" name="information" placeholder="information" required>
+                  <input value="{{organization.information}}" type="text" class="input-bg-gray form-control" name="information" placeholder="information" required>
                 </div>
 
                 <div class="form-group">
                   <div class="row">
                     <div class="col-sm-12">
-                      <button type="submit" name="register-submit" id="submit" class="btn" value="Save">ADD ORGANIZATION</button>
+                      <button type="submit" name="register-submit" id="submit" class="btn" value="Save">ADD ORGANIzATION</button>
```
Comment (rodrigojv) [inline src/app/organizations/add/template.hbs]: Should be ORGANIZATION.

File: src/styles/main.scss [hunk 2]
```diff
@@ -329,7 +329,7 @@ div.fp-form-container {
   max-height: 100px;
   height: 100px;
   object-fit: contain;
-  background: white url('/static/images/image-loader.gif') no-repeat center;
+  //background: white url('/static/images/image-loader.gif') no-repeat center;
```
Comment (rodrigojv) [inline src/styles/main.scss]: Remove commented code.

File: src/app/organizations/add/template.hbs (hunk 1 above)
Comment (carlosbflores) [reply src/app/organizations/add/template.hbs]: Fixed. All "Organization" is being renamed to english spelling "Organisation" with S.

File: src/app/organizations/add/template.hbs [hunk 3]
```diff
@@ -6,7 +6,7 @@
         <div class="row">
           <div class="col-md-12">
             <div class="card-title">
-              <h4> Add an Organization
+              <h4> Add an Organisation
```
Comment (rodrigojv) [inline src/app/organizations/add/template.hbs]: "Organization"

File: src/app/organizations/add/view.js [hunk 4]
```diff
@@ -8,9 +8,13 @@ import utils from '../../utils';
 export default Mn.View.extend({
   template: Template,
   events: {
+    'click #image-logo': 'selectFile',
+    'click #image-logo-text': 'selectFile',
```
Comment (rodrigojv) [inline src/app/organizations/add/view.js]: Maybe we can replace this by a single `click #image-cam: selectFile`

**Pull request merged**